### PR TITLE
Media: Enable row-based gallery in Uploads tab

### LINKED ISSUE #3319: Enable row based gallery in Uploads tab
## Task Description

Currently this feature is hidden behind a feature flag, `rowBasedGallery`. This issue is just to turn the feature flag on.

### PR DESCRIPTION
**Pull request by joannag6**
## Summary

Turns on feature where we use a row based gallery in the Uploads tab.

## User-facing changes

Gallery is now row based instead of column based.

## Testing Instructions

Verify that the usual operations work in this gallery arrangement as well. (eg. pagination, search, insert, edit, delete)

---

Fixes #3319

### CODE REVIEW

**Comment by joannag6** [comment]
Will merge this one once #3433 is merged.

**Review by diegovar — APPROVED** [review]

**Comment by diegovar** [comment]
Can this be merged now?

**Pull request merged**
